docs/5.community/4.contribution.md

```diff
@@ -139,15 +139,15 @@ Under projects with configuration as shown below, Corepack will install `v7.5.0`
 
 We use [ESLint](https://eslint.org) for both linting and formatting with [`@nuxt/eslint-config`](https://github.com/nuxt/eslint-config).
 
-##### IDE Setup
+#### IDE Setup
```

Comment:
I believe this falls under the [Use ESLint](https://nuxt.com/docs/community/contribution#use-eslint) section, hence 5 levels of nesting.

Reply:
5 levels doesn't seem to be able to jump, is this a problem on the Nuxt Content side?

> I believe this falls under the Use ESLint

Part of the ESLint section would be desirable. I agree with you on same this.

Reply:
fix this commit: 5d6c1231c8e9fe965504687b4931574ab51ff15f